docs/source/interfaces.rst

```diff
@@ -154,12 +154,12 @@ Video Interface
 -----------------------------------------
 Can be called the VI_STATUS_REG, or the VI_CONTROL REG, whichever you prefer.
 
-This register describes the format of the framebuffer in RDRAM, as well as enables and disables effects such as gamma, dithering, anti-aliasing, etc.
+This register describes the format of the frame buffer in RDRAM, as well as enables and disables effects such as gamma, dithering, anti-aliasing, etc.
```

Comment:
remove the space in the middle of the word "framebuffer" please, here and everywhere else